learning-labs/modules/local-development/pages/2-run-jenkins.adoc

```diff
@@ -62,7 +64,11 @@ If port 8080 is already in use by another process then this command will fail.
 
 [IMPORTANT]
 ====
+Windows OS: When using Docker for Windows, use the following command to run the Jenkins container: <br />
+docker run --name jenkins -v /usr/local/bin/docker:/usr/bin/docker -v //var/run/docker.sock:/var/run/docker.sock --privileged --user root -p 50000:50000 -p 8080:8080 -d jenkins/jenkins:lts
```

Comment:
did you mean to have two forward slashes when mounting the docker socket?

Reply:
Yes, the two forward slashes are correct. I believe it has to do with how docker is setup on Windows to properly mount the deamon.